test/NuGet.Clients.FuncTests/NuGet.MSSigning.Extensions.FuncTest/Commands/MSSignCommandTests.cs

```diff
@@ -24,6 +24,8 @@ namespace NuGet.MSSigning.Extensions.FuncTest.Commands
     public class MSSignCommandTests
     {
         private readonly string _noTimestamperWarningCode = NuGetLogCode.NU3002.ToString();
+        private readonly string _invalidCertificateFingerprintCode = NuGetLogCode.NU3043.ToString();
+        private const string _sha1Hash = "89967D1DD995010B6C66AE24FF8E66885E6E03A8";
```

Comment:
I think these break an IDE style rule that `static` and `const` fields shouldn't be `_`-prefixed.  Only instance fields should be `_`-prefixed.

`_noTimestamperWarningCode` -> `NoTimestamperWarningCode`
`_invalidCertificateFingerprintCode` -> `InvalidCertificateFingerprintCode`
`_sha1Hash ` -> `Sha1Hash`

Reply:
I renamed `_sha1Hash` -> `Sha1Hash`

`_noTimestamperWarningCode` and `_invalidCertificateFingerprintCode` are read-only strings. IDE analyzers warned of the lack of the prefix _ if I remove the prefix.